README.md

```diff
@@ -2,7 +2,8 @@
 
 [![Build Status](https://travis-ci.com/ThreeSixtyGiving/datastore.svg?branch=master)](https://travis-ci.com/ThreeSixtyGiving/datastore)
 
-
+[![Coverage Status](https://coveralls.io/repos/github/ThreeSixtyGiving/Dashboard/badge.svg?branch=master)](https://coveralls.io/github/ThreeSixtyGiving/Dashboard?branch=master)
+   
```

Comment:
Just looking at the preview of the README, I think it might be better to have these all on one line as it is shunting the actual content down the page.